pages/common/curl.md

```diff
@@ -5,32 +5,40 @@
 
 - Download a URL to a file:
 
-`curl "{{URL}}" -o {{filename}}`
+`curl {{http://example.com}} -o {{filename}}`
+
+- Download file using remote name:
+
+`curl -O {{http://example.com/filename}}`
+
+- Automatically continue/resume previous file transfer:
```

Comment:
This is one of the main uses I have for curl. Also, the [`tldr wget`](https://github.com/tldr-pages/tldr/blob/master/pages/common/wget.md) page has the analogous option: "Continue an incomplete download".